code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -67,6 +77,14 @@
 	build_path = /obj/item/reagent_containers/food/condiment/flour
 	category = list("initial","Food")
 
+/datum/design/rice_sack
```

Comment:
Idem, rice plants exist for a purpose.
In fact I'm going to revert the flour changes for the same reason.